.github/workflows/lint.yml

```diff
@@ -0,0 +1,27 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install --upgrade tox
```

Comment:
```
      python -m pip install --upgrade black mypy tox 
   - name: Format with black
      if: matrix.python-version >= 3.6
      run: |
        pip install black
        black --check . || true
    - name: Type check with mypy
      if: matrix.python-version >= 3.5
      run: |
        pip install mypy
        mypy --ignore-missing-imports .
```

Reply:
Black is already run via tox and I'm not using mypy here. But thanks!